services/director-v2/tests/integration/02/test__dynamic_sidecar_nodeports_integration.py

```diff
@@ -256,6 +258,9 @@ async def fast_api_app(
     monkeypatch.setenv("DIRECTOR_V2_CELERY_SCHEDULER_ENABLED", "false")
     monkeypatch.setenv("DYNAMIC_SIDECAR_TRAEFIK_ACCESS_LOG", "true")
     monkeypatch.setenv("DYNAMIC_SIDECAR_TRAEFIK_LOGLEVEL", "debug")
+    # patch host for dynamic-sidecar, not reachable via
+    # docker container to container networking otherwise
+    monkeypatch.setenv("RABBIT_HOST", f"{get_ip()}")
```

Comment:
It is required because this IP is sent to the dynamic-sidecar (running inside a container), it cannot use "localhost" which is normally provided by `rabbit_service `.